recipes/stages/_base_/data/multicls_data.py

```diff
@@ -0,0 +1,10 @@
+_base_ = [
```

Comment:
I can't find the usage of this file. Why is this file for?

Reply:
Yes, this file was used for different augmentations setup. According to the last experiments there is no need to change augmentations. So, I will delete this file.